src/main/java/org/sefglobal/scholarx/model/Program.java

```diff
@@ -35,7 +35,7 @@
     @Enumerated(EnumType.STRING)
     @Column(length = 20,
             nullable = false)
-    private ProgramState state;
+    private ProgramStatus state;
```

Comment:
```suggestion
    private ProgramStatus status;
```

Reply:
Changed the name of the variables too as indicated and also in the methods